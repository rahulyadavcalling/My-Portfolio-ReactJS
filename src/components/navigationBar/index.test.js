import React from "react";
import renderer from "react-test-renderer";
import { BrowserRouter as Router } from "react-router-dom";

import NavBar from "./index.js";

it("renders correctly when there are no items", () => {
  const tree = renderer
    .create(
      <Router>
        <NavBar />
      </Router>
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
