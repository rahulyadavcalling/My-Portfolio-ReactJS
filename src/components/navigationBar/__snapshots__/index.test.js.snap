// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly when there are no items 1`] = `
<nav
  className="navbar-default navbar navbar-inverse navbar-fixed-top"
  style={
    Object {
      "backgroundColor": "transparent",
      "boxShadow": null,
    }
  }
>
  <div
    className="container"
  >
    <div
      className="container-fluid"
    >
      <div
        className="navbar-header"
      >
        <button
          className="navbar-toggle collapsed"
          onClick={[Function]}
          type="button"
        >
          <span
            className="sr-only"
          >
            Toggle navigation
          </span>
          <span
            className="icon-bar"
          />
          <span
            className="icon-bar"
          />
          <span
            className="icon-bar"
          />
        </button>
        <span
          className="navbar-brand"
        >
          Welcome
        </span>
      </div>
      <div
        aria-expanded={null}
        className="navbar-collapse collapse"
      >
        <ul
          className="nav navbar-nav navbar-center nav navbar-nav"
          role={null}
        >
          <li
            className="sc-bdVaJa dxrIjc"
            role="presentation"
            style={undefined}
          >
            <a
              href="#"
              isselected="true"
              onClick={[Function]}
              onKeyDown={[Function]}
              role="button"
            >
              Welcome
            </a>
          </li>
          <li
            className="sc-bdVaJa jBEbvB"
            role="presentation"
            style={undefined}
          >
            <a
              href="#"
              isselected="false"
              onClick={[Function]}
              onKeyDown={[Function]}
              role="button"
            >
              About Me
            </a>
          </li>
          <li
            className="sc-bdVaJa jBEbvB"
            role="presentation"
            style={undefined}
          >
            <a
              href="#"
              isselected="false"
              onClick={[Function]}
              onKeyDown={[Function]}
              role="button"
            >
              Skills
            </a>
          </li>
          <li
            className="sc-bdVaJa jBEbvB"
            role="presentation"
            style={undefined}
          >
            <a
              href="#"
              isselected="false"
              onClick={[Function]}
              onKeyDown={[Function]}
              role="button"
            >
              Projects
            </a>
          </li>
          <li
            className="sc-bdVaJa jBEbvB"
            role="presentation"
            style={undefined}
          >
            <a
              href="#"
              isselected="false"
              onClick={[Function]}
              onKeyDown={[Function]}
              role="button"
            >
              Resume
            </a>
          </li>
        </ul>
      </div>
    </div>
  </div>
</nav>
`;
